schemas/kube-lint-config.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "KubeLinter Configuration",
+    "type": "object",
+    "properties": {
+        "checks": {
+            "type": "object",
+            "description": "Configure which built-in checks to run or skip",
+            "properties": {
+                "doNotAutoAddDefaults": {
+                    "type": "boolean",
+                    "description": "Disable all built-in checks"
+                },
+                "addAllBuiltIn": {
+                    "type": "boolean",
+                    "description": "Enable all built-in checks. Takes precedence over doNotAutoAddDefaults"
+                },
+                "ignorePaths": {
+                    "type": "array",
+                    "description": "File or directory globs to skip. Uses https://pkg.go.dev/github.com/bmatcuk/doublestar#Match syntax",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "include": {
+                    "type": "array",
+                    "description": "List of specific checks to run. Exclude always takes precedence over include",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "exclude": {
+                    "type": "array",
+                    "description": "List of specific checks to skip (takes precedence over include)",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            },
+            "additionalProperties": false
+        },
+        "customChecks": {
+            "type": "array",
+            "description": "User-defined checks based on templates",
+            "items": {
```

Comment:
If I read the file at `https://github.com/stackrox/kube-linter/blob/main/pkg/config/check.go` correctly, there is one more parameter for the `customeChecks` array: `description`. Would you mind adding it ?